zthin-parts/zthin/bin/refresh_bootmap

```
@@ -84,11 +84,15 @@ function cleanup_fcpdevices {
   inform "Begin to disconnect FCPs"
   for i in "${!fcps[@]}"
   do
-    chccwdevDevice ${fcps[$i]} "offline"
-    for j in "${!wwpns[@]}"
-    do
-      disconnectFcp ${fcps[$i]} 0x${wwpns[$j]} ${lun}
-    done
+    if [[ -z "$wwpns" ]];then
+      /opt/zthin/bin/smcli Image_Device_Undedicate -T $zthinUserID -v ${fcps[$i]} &> /dev/null
```

I didn't check previous discussion, we can safely ignore the error here?

---

or at least print something to know what really happened?